packages/app-accounts/src/modals/Forgetting.tsx

```diff
@@ -61,7 +60,13 @@ class Forgetting extends React.PureComponent<Props> {
           {t('Confirm account removal')}
         </Modal.Header>
         <Modal.Content>
-          <AddressSummary value={address} />
+          <AddressRow
+            isInline
+            value={address}
+          >
+            <p>{t('You are about to remove this account from your list of available accounts. Once completed, should you need access again, you need to re-create the account either via seed or via a backup file.')}</p>
+            <p>{t('This operaion does not remove the history of the account from the chain, nor any associated funds from the account. The forget operation only limits your access to the account on this local machine.')}</p>
```

Comment:
Should we talk about browser instead?
```suggestion
            <p>{t('This operaion does not remove the history of the account from the chain, nor any associated funds from the account. The forget operation only limits your access to the account on this browser.')}</p>
```